lib/modules/manager/github-actions/__snapshots__/extract.spec.ts.snap

```diff
@@ -9,6 +9,9 @@ exports[`modules/manager/github-actions/extract extractPackageFile() extracts mu
     "datasource": "github-tags",
     "depName": "actions/bin",
     "depType": "action",
+    "registryUrls": [
+      "https://github.com",
```

Comment:
I expected to see non-github.com registryUrls added. github.com remains the default, right - so this addition is not necessary?

Reply:
Correct, the addition is not necessary, but I added it to give the control over the registryUrls to the manager and provide them explicitly.

The alternative, and happy to add that, would be a logic where if we run against github.com we don't provide any registryUrls and if we run against Enterprise Server we use `registryUrls: [github.enterprise.com, github.com]`. 

From my observation the only difference would be if `defaultRegistryUrl` is honored or not.

Reply:
Changed the logic to only set them if they need to modified in 58fe56830